pandas/tests/indexes/multi/test_analytics.py

```diff
@@ -348,9 +348,9 @@ def test_numpy_ufuncs(idx, func):
 )
 def test_numpy_type_funcs(idx, func):
     msg = (
-        "ufunc '{}' not supported for the input types, and the inputs"
-        " could not be safely coerced to any supported types according to"
-        " the casting rule ''safe''"
-    ).format(func.__name__)
+        f"ufunc '{func.__name__}' not supported for the input types, and the inputs"
+        f" could not be safely coerced to any supported types according to"
+        f" the casting rule ''safe''"
```

Comment:
...and here

Reply:
Yep, tested it in my python interpreter and it works.  Thanks for the suggestions, @WillAyd